### PR TITLE
*(ticdc): enable new scheduler by default

### PR DESCRIPTION
**Pull request by liuzix**
### What problem does this PR solve?

Issue Number: ref #4126

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
Enables a new scheduler.
```

### CODE REVIEW

File: cdc/owner/changefeed_test.go [hunk 1]
```diff
@@ -138,6 +138,9 @@ func createChangefeed4Test(ctx cdcContext.Context, t *testing.T) (*changefeed, *
 	}, func() DDLSink {
 		return &mockDDLSink{}
 	})
+	cf.newScheduler = func(ctx cdcContext.Context, startTs uint64) (scheduler, error) {
+		return newSchedulerV1(), nil
+	}
```
Comment (amyangfei) [inline cdc/owner/changefeed_test.go:143]: Is this test only available with old scheduler?

**Review by amyangfei — APPROVED** [review]

File: cdc/owner/changefeed_test.go (hunk 1 above)
Comment (liuzix) [reply cdc/owner/changefeed_test.go:143]: The correctness of the scheduler is not the goal of these tests. The old scheduler is like a "mock scheduler" here.

File: pkg/cmd/server/server_test.go [hunk 2]
```diff
@@ -179,7 +179,8 @@ func TestParseCfg(t *testing.T) {
 			TableActor: &config.TableActorConfig{
 				EventBatchSize: 32,
 			},
-			EnableDBSorter: false,
+			EnableDBSorter:     false,
+			EnableNewScheduler: true,
```
Comment (overvenus) [inline pkg/cmd/server/server_test.go:183]: Could you remove `EnableNewScheduler` in debug.go ?

https://github.com/pingcap/tiflow/blob/92fc4c4593cafe3d81936de3f4eb5775683a8bae/pkg/config/debug.go#L35

**Review by overvenus — APPROVED** [review]

**Pull request merged**